finance-api/api-reference.md

```diff
@@ -0,0 +1,282 @@
+
+* [About Finance API](#about-finance-api)
+  * [Base URL](#base-url)
+  * [OAuth scope](#oauth-scope)
+* [Fetch account balance](#fetch-account-balance)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Fetch account transactions](#fetch-account-transactions)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Fetch payout information](#fetch-payout-information)
+  * [Parameters](#parameters)
+  * [Responses](#responses)
+* [Examples](#examples)
+  * [Fetch balance for a liquid account](#fetch-balance-for-a-liquid-account)
+  * [Fetch transactions for a liquid account](#fetch-transactions-for-a-preliminary-account)
+  * [Fetch payout information on a specific day](#fetch-payout-information-on-a-specific-day)
+* [Related resources](#related-resources)
+* [Related API reference](#related-api-reference)
+  
+### Base URL
+https://finance.zettle.com
+
+### OAuth Scope
+`READ:FINANCE`
+<!-- For more information on how to get authorisaition for the scope, see See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) -->
+
+## Fetch account balance
+Returns the balance that is available in a merchant's Zettle preliminary or liquid account at a specific time.
+
+If the balance is zero, it means the following:
+
+* For the preliminary account, the money is moved from the merchant's preliminary account to their liquid account.
+* For the liquid account, the money is paid out from Zettle to the mechant's bank account.
+<!-- Returns accumulated balance at a specific point in time. After a deposit has been made from LIQUID account to merchant’s bank account (daily, monthly or weekly), the balance is usually zero. If not, it will return the amount that is still due in merchant’s Zettle account. -->
+
+```
+GET /organizations/{organizationUuid}/accounts/{accountTypeGroup}/balance
+```
+
+See example [Fetch balance for a liquid account](#fetch-balance-for-a-liquid-account).
+
+### Parameters
+
+<details open="true">
+<summary>Parameters</summary>
+
+|Name |Type |In |Required/Optional |Description
+|:---- |:---- |:---- |:---- |:----
+|organizationUuid |string |path |required |Unique identifier for your organization. You can specify the value with one of the following: <br/><ul><li> Use `self` as the value. This will retrieve your organizationUuid from the authentication token in the request.</li><li> Get it by using the https://oauth.izettle.com/users/me endpoint of OAuth2 API. See [OAuth2 API](https://github.com/iZettle/api-documentation/blob/master/authorization.adoc) for more information.</li></ul> 
+|accountTypeGroup |string |path |required |The account type from which the data like transactions is retrieved. You can use one of the following account types: <br/><ul><li> `PRELIMINARY` account where transactions are to be confirmed by third-party acquirers.</li><li> `LIQUID` account where transactions are to be paid out to the merchant.</li></ul>
```

Comment:
> The account type from which the data like transactions is retrieved.

I don't think we should keep this sentence. The endpoint is balance not transactions. It's hard to understand.